azure/sdk-identity-resources-storage/README.md

```diff
@@ -0,0 +1,71 @@
+﻿## Prerequisites
+
+* An [Azure subscription]().
+
+## Preparing to run the sample
+
+Use the [Azure Cloud Shell](https://shell.azure.com) to create/get client secret credentials.
+
+* Create a service principal and configure its access to Azure resources:
+
+    ```bash
+    az ad sp create-for-rbac -n <your-application-name> --skip-assignment
+    ```
+
+    Output:
+
+    ```json
+    {
+        "appId": "generated-app-ID",
+        "displayName": "dummy-app-name",
+        "name": "http://dummy-app-name",
+        "password": "random-password",
+        "tenant": "tenant-ID"
+    }
```

Comment:
```suggestion
    {
        "appId": "generated-app-ID",
        "displayName": "sample-app-name",
        "name": "http://sample-app-name",
        "password": "random-password",
        "tenant": "tenant-ID"
    }
```

Instead of "dummy", use "sample"